pom.xml

```diff
@@ -38,90 +39,43 @@
   </licenses>
   
   <properties>
-    <!-- QueueItemAuthenticator is since 1.520 -->
-    <jenkins.version>1.532</jenkins.version>
-    <jenkins-test-harness.version>1.532</jenkins-test-harness.version>
-    <java.level>6</java.level>
+    <jenkins.version>2.346.3</jenkins.version>
   </properties>
   
   <dependencies>
     <dependency>
-      <!-- The one depended by Jenkins 1.532 to avoid NoSuchMethodError of jenkins-test-harness -->
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.1</version>
-      <scope>test</scope>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>matrix-auth</artifactId>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>cloudbees-folder</artifactId>
-      <version>4.0</version>
-      <scope>test</scope>
+      <artifactId>matrix-project</artifactId>
```

Comment:
Note: `MatrixProject` is used here: https://github.com/jenkinsci/jobcopy-builder-plugin/blob/jobcopy-builder-1.4.2/src/main/java/jp/ikedam/jenkins/plugins/jobcopy_builder/JobcopyBuilder.java#L354
and this cannot be optional.